server/rest/src/test/java/org/infinispan/rest/RestOperationsOffHeapTest.java

```diff
@@ -0,0 +1,16 @@
+package org.infinispan.rest;
+
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.cache.StorageType;
+import org.testng.annotations.Test;
+
+@Test(groups = "functional", testName = "rest.RestOperationsOffHeapTest")
+public class RestOperationsOffHeapTest extends RestOperationsTest {
+
+   @Override
+   public ConfigurationBuilder getDefaultCacheBuilder() {
```

Comment:
It is worth to run whole test suite (there are many tests in `RestOperationsTest`) just for one scenario? 

Maybe it's worth to write a single test, that would call PUT and check cache content?

Reply:
> It is worth to run whole test suite (there are many tests in RestOperationsTest) just for one scenario?

Definitely. During a simple put and get does not cover the whole spectrum of features, take for example ISPN-8370 that was just created